config.go

```diff
@@ -70,16 +71,53 @@ func (cfg Config) MakeLabels(types []string) string {
 }
 
 // GetNativePropertyValue is called during building properties for save and when the expanded property key exists in the config.
-func (cfg Config) GetNativePropertyValue(node *lpg.Node, expandedPropertyKey, val string) interface{} {
-	if _, exists := cfg.PropertyTypes[expandedPropertyKey]; exists {
-		va := ls.GetValueAccessor(cfg.PropertyTypes[expandedPropertyKey])
-		native, err := va.GetNativeValue(val, node)
+func (cfg Config) GetNeo4jPropertyValue(expandedPropertyKey string, val string) (interface{}, error) {
+	propType, exists := cfg.PropertyTypes[expandedPropertyKey]
+	if !exists {
+		return val, nil
+	}
+	var v interface{}
+	var err error
+	switch propType {
+	case "Integer":
+		v, err = strconv.Atoi(val)
+		if err != nil {
+			return nil, err
+		}
+	case "Float":
+		v, err = strconv.ParseFloat(val, 64)
+		if err != nil {
+			return nil, err
+		}
+	case "Boolean":
+		v, err = strconv.ParseBool(val)
+		if err != nil {
+			return nil, err
+		}
+	case "Date":
+		t, err := time.Parse("2006-01-02", val)
```

Comment:
Get the format to parse from the config:

propertyName: Date, MM/YY/DDDD

parse the propType to find the format. Use goment library to parse using the format